http/oidc/src/main/java/org/wildfly/security/http/oidc/OidcClientConfiguration.java

```diff
@@ -126,6 +132,17 @@ public enum RelativeUrlsUsed {
     protected boolean verifyTokenAudience = false;
 
     protected String tokenSignatureAlgorithm = DEFAULT_TOKEN_SIGNATURE_ALGORITHM;
+    protected String authenticationRequestFormat;
+    protected String requestObjectSignatureAlgorithm;
```

Comment:
It's good to be consistent with the JSON config option that will be used to configure this. It looks like we use signing there as opposed to signature, right? If so, we should update this.